daemon/src/projection.rs

```diff
@@ -1165,8 +1181,8 @@ impl Actor {
         );
     }
 
-    fn handle(&mut self, msg: Update<Option<model::MakerOffers>>) {
-        self.tx.send_order_update(msg.0);
+    fn handle(&mut self, msg: Update<(ContractSymbol, Option<model::MakerOffers>)>) {
+        self.tx.send_order_update(msg.0 .0, msg.0 .1.into());
```

Comment:
🔧

```suggestion
        self.tx.send_order_update(msg.0.0, msg.0.1.into());
```

Reply:
dprint is not complaining, so maybe it's expected?

Reply:
jap, it's dprint.